CONTRIBUTING.md

```diff
@@ -3,5 +3,15 @@
 - Install the recommended extensions for automatic formatting on save.
 - We use the [ava](https://github.com/avajs/ava) test runner. To run one or more specific test(s), use `npm run test -- -m <title>`.
 - The `codeql` executable must be on the path before running any tests.
-- All compiled artifacts and `node_modules` dependencies should be checked in.
```

Comment:
Should we still mention that compiled artifacts should be checked in? The files in `dist` should still be checked in.

Reply:
Oh yes we still need the build artifacts! 🤦‍♀️